patternfly/patternfly-elements
# feat: Move the generator into the repo

## Moving the generator into the repo

* Michael and I discussed the possibility of moving the generator from a separate repo into the monorepo, this PR accomplishes that

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Moving the generator from a separate repo into this one

The benefit of moving the generator into the monorepo is that when updates occur in the parent project, it's easier to ensure the generator is updated at the same time.  This creates a "source of truth" for building components.

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. `npm run new` should generate prompts

#### Requirements

Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
